docs/design/agents.md

```
@@ -613,29 +613,35 @@ does.
+    end
+    subgraph S4[Guards]
+        G1[RateLimitGuard]
+        G2[ReviewGateGuard]
+        G3[RollbackGuard]
+        G4[ShadowEvaluationGuard]
+        G5[CompositeGuard]
+    end
+    subgraph S5[Adapter.apply]
+        A1[IdentityAdapter]
+        A2[StrategySelectionAdapter]
+        A3[PromptTemplateAdapter]
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)